games/game_witcher3.py

```diff
@@ -11,20 +15,32 @@ class Witcher3Game(BasicGame):
 
     Name = "Witcher 3 Support Plugin"
     Author = "Holt59"
-    Version = "1.0.0a"
-    Description = "The Description Of The Dead"
+    Version = "1.1.0"
+    Description = "The Description Of The Dead| Updated to work with GOG by Luca/EzioTheDeadPoet"
```

Comment:
This description is a left-over from when I was testing the plugin and should be removed. If you want to add yourself, please use the `Author` field:

```python
Author = "Holt59 & Luca/EzioTheDeadPoet"
```